test/client/fixtures/automation/click-test.js

```diff
@@ -207,6 +207,51 @@ $(document).ready(function () {
             });
     });
 
+    asyncTest('an active input should be blurred and a parent of a disabled input should be focused after a click the disabled input', function () {
```

Comment:
@VasilyStrelyaevm could you check the test name please?

Reply:
```
    asyncTest('an active input should be blurred and a parent of a disabled input should be focused after a click on the disabled input', function () {
```

Reply:
I don't usually check test names, is this one a very important test?

Reply:
No, sorry, just felt that it was something wrong with the test name.

Reply:
that's ok, I'm not against checking test names